components/brave_wallet_ui/stories/locale.ts

```diff
@@ -429,12 +429,15 @@ provideStrings({
   braveWalletBuyRampNetworkName: 'Ramp.Network',
   braveWalletBuyWyreName: 'Wyre',
   braveWalletBuySardineName: 'Sardine',
+  braveWalletBuyTransakName: 'Transak',
   braveWalletBuyRampDescription: 'Buy with CC/Debit or ACH. Competitive Rates.',
   braveWalletBuyWyreDescription: 'Buy with CC/Debit or ACH. Competitive Rates.',
   braveWalletBuySardineDescription: 'Instant buy with your bank account. Lower fees.',
+  braveWalletBuyTransakDescription: 'Instant buy with your bank account. Lower fees.',
```

Comment:
Is duplicate of `braveWalletBuySardineDescription`

Reply:
@Douglashdaniel It is okay because the on-ramp can request their description to be changed in the future so we need a separate string for each.